frontend/android/feature-notification-usage/src/test/kotlin/net/matsudamper/money/frontend/android/feature/notificationusage/NotificationUsageAutoAddProcessorTest.kt

```
@@ -0,0 +1,203 @@
+    },
+)
+
+private class AutoAddFakeAppSettingsRepository(
+    autoAddEnabledByFilterId: Map<String, Boolean>,
+) : AppSettingsRepository {
+    override val showImagesInMonthlyScreen: Flow<Boolean> = MutableStateFlow(false)
+    private val notificationUsageAutoAddEnabledFlows = autoAddEnabledByFilterId
+        .mapValuesTo(mutableMapOf()) { (_, enabled) -> MutableStateFlow(enabled) }
+
+    override fun setShowImagesInMonthlyScreen(value: Boolean) {
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)